server/services/offendersService.js

```diff
@@ -1205,6 +1205,13 @@ module.exports = function createOffendersService(
           recordExists: !!foundCatRecord,
           approvalDateDisplay: dateConverter(nomisRecord.approvalDate),
           sequence: foundCatRecord && foundCatRecord.sequence,
+          tprsSelected:
+            foundCatRecord &&
+            foundCatRecord.formObject &&
+            foundCatRecord.formObject.openConditions &&
+            foundCatRecord.formObject.openConditions.tprs &&
+            foundCatRecord.formObject.openConditions.tprs.tprsSelected &&
+            foundCatRecord.formObject.openConditions.tprs.tprsSelected === 'Yes',
```

Comment:
I think this could use a combo of optional chaining or a fallback to false to get the same outcome:

```
foundCatRecord?.formObject?.openConditions?.tprs?.tprsSelected === 'Yes' || false
```

Reply:
Agree, looks nicer. In order to spare on the time the integration tests are running, can we do this as an improvements on your branch?